src/helm/proxy/clients/palmyra_client.py

```diff
@@ -144,7 +180,49 @@ def do_it():
         )
 
     def tokenize(self, request: TokenizationRequest) -> TokenizationRequestResult:
-        raise NotImplementedError("Use the HuggingFaceClient to tokenize.")
+        cache_key = asdict(request)
+
+        try:
+
+            def do_it():
+                _initialize_sentence_piece_processor()
+                if request.encode:
+                    tokens = _sentence_piece_processor.EncodeAsIds(request.text)
+                else:
+                    tokens = [
+                        piece.replace("▁", " ") for piece in _sentence_piece_processor.EncodeAsPieces(request.text)
```

Comment:
Do we have a utility function for this?

Reply:
The utility function is in an open PR #1549 so I can't use it yet. Added a TODO.